### ISSUE TITLE
1.3.0 does not install on Ubuntu

### ISSUE THREAD

**Issue by tay-bird** [newrelic/infra-integrations#17]
##### Description of the problem

We have started seeing the following failure on our machines:

```$ sudo apt-get install newrelic-infra-integrations
Reading package lists... Error!
E: Problem parsing dependency Replaces
E: Error occurred while processing newrelic-infra-integrations (NewVersion2)
E: Problem with MergeList /var/lib/apt/lists/download.newrelic.com_infrastructure%5fagent_linux_apt_dists_xenial_main_binary-amd64_Packages
E: The package lists or status file could not be parsed or opened.
```

These failures began at the same time as this release: https://github.com/newrelic/infra-integrations/releases/tag/v1.3.0

It seems that @areina replaced the previous v1.3.0 tag, which was pushed out last week. Not sure why that happened, but it seems to be causing this issue.

##### Infrastructure Integration

- DEB package

##### OS

- [ ] Debian, version 7 ("Wheezy") or higher
- [ ] Ubuntu, versions 12.04, 14.04, and 16.04 (LTS  versions)

**Comment by areina** [comment]
Hello!

TL;DR: We identified the problem and the issue should be already fixed. Could you let us know if it's working well for you, please? Sorry for the inconvenience.

Yes, we've had an issue with the metadata of our package v1.3.0. We tagged the code the past week but the packages (deb/rpm) haven't been released until today. After the release we've discovered the problem and immediately we've start fixing it. The issue was with the generation of the package and that's why I've needed to recreate the git tag instead of creating a new version. That have allowed us to generate a new package with the proper metadata. 

Again, we are sorry for the inconvenience.

**Comment by tay-bird** [comment]
It seems like this affects machines where the defective 1.3.0 package was installed previously. The solution is to run `apt update` to clear the cache.
